Graphics/Implicit/Definitions.hs

```diff
@@ -304,7 +308,7 @@ instance Show SymbolicObj2 where
     Circle r         -> showCon "circle"     @| r
     Polygon ps       -> showCon "polygon"    @| ps
     Rotate2 v obj    -> showCon "rotate"     @| v     @| obj
-    Transform2 m obj -> showCon "transform2" @| m     @| obj
```

Comment:
so, you don't like "transform, transform3", but do like "rotate, rotate3"?

Reply:
Thanks, renamed to match `Rotate / Rotate3`.

We might also get rid of Rotate, Rotate3 and Translate and implement these using Transform/Transform3 as these are easier to compose (just a matrix multiplication), so we don't have redundant code paths to do the same thing. On a second thought, they wouldn't `Show` nicely so NO. Better to do that in canonicalization pass which won't affect `Show` of user defined trees.

Reply:
The naming was consistent, just the `showCon` param was outputting transform2 instead of transform (`transform = Transform2`). Restored.